src/config/defaults.ts

```diff
@@ -5,6 +5,9 @@ export const defaults: Config = {
   serverURL: '',
   defaultDepth: 2,
   maxDepth: 10,
+  defaultMaxTextLength: 500,
+  defaultMaxNum: 9007199254740991, // 2^53 - 1 which is the MAX_SAFE_INTEGER for JS
```

Comment:
Question - is there any reason why anyone would want to change this? Say someone wants to make it a bigger number than this. Wouldn't that break JS anyway? 

Here is what I'm thinking. If our default is the `MAX_SAFE_INTEGER` size anyway, then why have it exposed as a config option? What would someone's use case be to reduce the max number size? Wouldn't they want to do this on a field by field basis anyway? Number fields with large values wouldn't ever be able to produce big (say 1TB) values as it is, would they?

If we do keep these new config values, should we enforce, maybe in `joi` schema validation, that the number passed is always less than the `MAX_SAFE_INTEGER` value?

Overall I think the intent of this PR was to restrict fields that accept text-based strings so that no one can send a huge string of data into the API. This would seem to apply to text fields, textarea, email, password, etc. and maybe even rich text but it applies less-so if JS already handles this for us with number fields.

This all being said I am not sure of the ramifications of `MAX_SAFE_INTEGER` as it is so you might know more than me here.

Reply:
I agree, I just realized after consulting the mongodb docs and tested Payload the true max integer the database can handle is 2^301 or some crazy number. Not enough to mess with the DB size. Will remove this.